specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/preview/2018-08-15-preview/timeseriesinsights.json

```diff
@@ -1201,12 +1202,56 @@
       ],
       "description": "The sku determines the capacity of the environment, the SLA (in queries-per-minute and total capacity), and the billing rate."
```

Comment:
Also determines the "storage type" now?

Reply:
I updated the description to indicate the Sku determines the type of environment.